test/hubbardoperators.jl

```diff
@@ -64,10 +64,15 @@ end
                       d_num(particle_symmetry, spin_symmetry) ≈
                       d_num(particle_symmetry, spin_symmetry) *
                       u_num(particle_symmetry, spin_symmetry)
+            else
+                @test_broken u_plus_u_min(particle_symmetry, spin_symmetry)
+                @test_broken d_plus_d_min(particle_symmetry, spin_symmetry)
```

Comment:
I might be noticing this a bit late, but is the goal here to check if this throws an error? `@test_broken` implies that you expect it to work, but it currently does not. `@test_throws ArgumentError` is the better way to check that something throws.

Do you mind checking this in the other tests as well?